backend/services/prisonerProfileService.ts

```diff
@@ -65,6 +83,19 @@ export default ({
 
     const systemContext = await systemOauthClient.getClientCredentialsTokens(username)
 
+    // TODO: Temporary measure to allow 3rd party MegaNexus time to implement caching and other techniques to their API so it can handle high volume of calls. To be refactored!
+    const getNeurodivergenceSupportNeed = async (agencyId) => {
+      if (canViewNeurodivergenceSupportData(agencyId)) {
+        const divergenceData = await Promise.all(
```

Comment:
As discussed - this is wrong. You are effectively wrapping the call result in an array.

You ought to just use the standard promise in this situation